_drgn.pyi

```diff
@@ -1622,6 +1635,24 @@ class TypeParameter:
     name: Optional[str]
     """Parameter name, or ``None`` if the parameter is unnamed."""
 
+class TypeTemplateParameter:
+    """
+    A ``TypeTemplateParameter`` represents a template parameter.
+
+    :param type: Type of the template parameter. This may be a :class:`Type` or ``None``.
+    :param name: Name of the template parameter. This may be ``None`` if the parameter is
+        unnamed.
```

Comment:
These should go in the docstring for `__init__()` now (and you can copy the text for `TypeMember.__init__()` since that correctly mentions that `type` can be a callable).